crates/vault-registry/src/tests.rs

```diff
@@ -1353,13 +1353,121 @@ mod get_vaults_with_issuable_tokens_tests {
             // update the exchange rate
             ext::oracle::backing_to_issuing::<Test>.mock_safe(move |x| MockResult::Return(Ok(x / 2)));
 
-            assert_err!(
-                VaultRegistry::get_vaults_with_issuable_tokens(),
-                TestError::NoVaultWithIssuableTokens
+            assert_eq!(VaultRegistry::get_vaults_with_issuable_tokens(), Ok(vec!()));
+        })
+    }
+}
+
+mod get_vaults_with_redeemable_tokens_test {
+    use super::*;
+
+    #[test]
+    fn get_vaults_with_redeemable_tokens_returns_empty() {
+        run_test(|| {
+            create_sample_vault();
```

Comment:
Why is this here?

Reply:
To have a vault with 0 redeemable tokens, and make sure it doesn't get returned.

Reply:
Makes sense, can you add that as a comment?